### ISSUE TITLE
Panning: Flickering

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#207]
```
GeoDa 1.7.45
Win-32: For panning, there's slight flickering before panning works. Not a 
problem in Mac where panning is instant.
```

Original issue reported on code.google.com by `jkoschin...@gmail.com` on 27 Jul 2015 at 12:36

**Comment by lixun910** [comment]
This should be a problem when you use windows in virtual machine. I've tested on native windows, and there is no flickering. 

Please recheck this issue.

**Comment by jkoschinsky** [comment]
JK: verify on Windows

**Comment by jkoschinsky** [comment]
GeoDa 1.7.89 (Win 7 Professional (64-bit)): It does still flicker before panning.
Both with mouse and touchpad.

**Comment by lixun910** [comment]
If a large dataset load, flicker is normal. From what I tested with NAT, no flicking.

**Comment by jkoschinsky** [comment]
GeoDa 1.7.91 (Win 7 Professional (64-bit))

It still flickers slightly with smaller sizes as well (e.g. balt and nat) but not bad enough to be blocker issue right now.

**Comment by jkoschinsky** [comment]
Xun fixed it! very cool

Verified fix in GeoDa 1.7.95 (Win 7 Professional (64-bit)).
